common/src/main/java/earth/terrarium/cadmus/common/claims/ClaimApiImpl.java

```diff
@@ -132,16 +134,17 @@ public boolean canInteractWithBlock(Level level, BlockPos pos, InteractionType t
         }
         return canAccess(level, pos, player, CadmusAutoCompletes.BLOCK_INTERACTIONS, ModGameRules.RULE_DO_CLAIMED_BLOCK_INTERACTIONS,
             (id, server) -> {
-                Block block = level.getBlockState(pos).getBlock();
-                if (block instanceof DoorBlock || block instanceof TrapDoorBlock) {
+                BlockState state = level.getBlockState(pos);
+                Block block = state.getBlock();
+                if (state.is(BlockTags.DOORS) || state.is(BlockTags.TRAPDOORS)) {
                     return AdminClaimHandler.getBooleanFlag(server, id, ModFlags.USE_DOORS);
                 }
 
-                if (block instanceof AbstractChestBlock<?>) {
+                if (block instanceof AbstractChestBlock<?> || state.is(Cadmus.FABRIC_CHESTS) || state.is(Cadmus.FORGE_CHESTS)) {
```

Comment:
Use a common cadmus:chest tag and just include the platform tags as optional tags within it